Adjust_dataset.py

```diff
@@ -0,0 +1,43 @@
+#Script changing labels of emotions to 1 - smile, 0 - no smile and saving changed dataset to new csv file
+
+import pandas as pd
+
+data_set = pd.read_csv("fer2013.csv")
+
+print("Before:")
```

Comment:
Please put the whole code into a function and call from to:
```
def main():
    your_stuff()

if` __name__ == "__main__":
    main()
```
Inspiration on why to do this:
https://stackoverflow.com/questions/419163/what-does-if-name-main-do